zulipterminal/cli/run.py

```diff
@@ -152,6 +158,11 @@ def main(options: Optional[List[str]]=None) -> None:
         prof = cProfile.Profile()
         prof.enable()
 
+    if args.version:
+        msg = 'zulip-terminal ' + ZT_VERSION
+        print(msg)
```

Comment:
Since we use both 'zulip-term' and 'zulip-terminal' names for the command/project, it might be simpler to just use 'version: ' here, since users must know the command they're running :) Now that you're not using msg more than once, I'd just combine the two lines together, too.

Reply:
@neiljp I think "Zulip Terminal ZT_VERSION" should be better. Would it also be good to include the path from where zulip-term is running? I should help users easy differentiate if they are running the dev version or released one.

Reply:
@amanagr , Wouldn't it be better to add the include path with #288 ?

Reply:
Hmm... maybe but I am not sure since git already means it's the dev version. Well I don't think it fits well with this PR as well.